### PR TITLE
Fix typo and syntax 

### PR DESCRIPTION
**Pull request by yegorich**
Fix a typo and add missing commas.

Also, delete a blank line at the end of the file and
use /**/ for comments consistently.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>

### CODE REVIEW

**Review by mcuee — APPROVED** [review]

**Comment by tormodvolden** [comment]
Thanks. But if we start on this file, maybe also include other low-hanging fruit, like "Example" with capital E in the same phrase? A full review of the whole file would be welcome.

**Comment by yegorich** [comment]
@tormodvolden done.

**Comment by tormodvolden** [comment]
Great, thanks. It is useful in this case to add "in comments" in the commit message to clarify this is "cosmetic" without affecting code. But we can add that when we merge it.

**Review by tormodvolden — APPROVED** [review]

**Comment by mcuee** [comment]
I think this can be safely merged now.

File: android/examples/unrooted_android.c [hunk 1]
```diff
@@ -270,7 +270,7 @@ static void print_device(libusb_device *dev, libusb_device_handle *handle)
 }
 
 
-// fileDescriptor = is the native File Descriptor obtained in Java and transfered to native over JNA for Example.
+// fileDescriptor = is the native File Descriptor obtained in Java and transferred to native over JNA, for example.
```
Comment (tormodvolden) [inline android/examples/unrooted_android.c:273]: We should probably use /* */ instead of // for consistency.

Comment (yegorich) [reply android/examples/unrooted_android.c]: Done.

**Pull request closed without merging**